app/services/idv/steps/send_link_step.rb

```diff
@@ -96,6 +96,7 @@ def form_submit
 
       def formatted_destination_phone
         raw_phone = permit(:phone)[:phone]
+        idv_session[:phone] = raw_phone
```

Comment:
Will we need something like this for the new combined upload step, so that `idv_session[:phone]` exists for `link_sent.html.erb` in that flow?

Reply:
+1 would be nice to add the side effect in another spot, because this sounds like a getter

Reply:
also QQ on the name here, when we save a phone property to eventually by the IAL2 phone, is this the same one? I would call this `:phone_for_mobile_flow` or something if we wanted to clarify + differentiate

Reply:
👀 and will be reviewing with @eric-gade presently